gooddata-java/src/test/groovy/com/gooddata/sdk/service/export/ExportServiceTest.groovy

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2007-2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.service.export
+
+import com.gooddata.sdk.model.md.ProjectDashboard
+import com.gooddata.sdk.model.md.report.Report
+import com.gooddata.sdk.model.md.report.ReportDefinition
+import com.gooddata.sdk.service.GoodDataEndpoint
+import com.gooddata.sdk.service.GoodDataSettings
+import org.springframework.web.client.RestTemplate
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+import static com.gooddata.util.ResourceUtils.readObjectFromResource
+
+class ExportServiceTest extends Specification {
```

Comment:
There are no unit tests with valid arguments, e.g. for testing interaction with rest template. Create new github issue for this or add them in this PR.

Reply:
As there was this test before I only migrated it to Spock